compiler-plugin/src/main/kotlin/arrow/meta/higherkind/HigherKindPlugin.kt

```diff
@@ -54,9 +56,11 @@ private val KtClass.kindAritySuffix: String
 private val KtClass.partialKindAritySuffix: String
   get() = (arity - 1).let { if (it > 1) "$it" else "" }
 
+fun KtClass.isKinded() =
```

Comment:
All these functions are unrelated to autofold and should be moved to the KtUtils.

Reply:
Also this one should be named `isKind` since isKinded sounds like when a type is a kind where as this is exactly checking if this is the Kind type.